Database/Document.ts

```diff
@@ -23,4 +23,31 @@ export namespace Document {
 		const { id, created, changed, purged: purged, ...remainder } = document
 		return [{ id, created, changed, purged }, remainder as T]
 	}
+	export function update<T extends Record<string, any> = Document>(original: T, appendee: Partial<T>): T | undefined {
+		return JSON.parse(JSON.stringify({ ...original, ...appendee }))
+	}
+	export function append<T extends Record<string, any> = Document>(
```

Comment:
The `append` function could use another test that show how the function adds objects, arrays, etc.

Reply:
https://github.com/utily/cloudly-storage/pull/42

Reply:
Fantastic